src/lib/validation.js

```diff
@@ -40,7 +39,7 @@ const isFileInWorkspace = (error) => {
     filePath = filePath + '.module';
   }
 
-  return fs.existsSync(path.resolve(dirToActiveFile, filePath));
+  return vscode.workspace.fs.stat(path.resolve(dirToActiveFile, filePath));
```

Comment:
Roughly how to address `fs`, however this extension relies on `cli-lib` which uses `fs` directly